apps/web/pages/api/user/avatar.ts

```diff
@@ -30,39 +35,28 @@ async function getIdentityData(req: NextApiRequest) {
         id: orgId,
       }
     : org
-    ? getSlugOrRequestedSlug(org)
+    ? whereClauseForOrgWithSlugOrRequestedSlug(org)
     : null;
 
+  logger.debug({
+    org,
+    orgQuery,
+    username,
+    teamname,
+    currentOrgDomain,
+    isValidOrgDomain,
+    orgSlug,
+  });
+
   if (username) {
-    let user = await prisma.user.findFirst({
+    const user = await prisma.user.findFirst({
       where: {
         username,
         organization: orgQuery,
       },
       select: { avatar: true, email: true },
     });
 
-    /**
-     * TEMPORARY CODE STARTS - TO BE REMOVED after mono-user schema is implemented
```

Comment:
We don't need this hack now as the avatar itself has the orgId being passed from all places on booking pages.|
Goal is to do that from everywhere.